modules/recieve_statustxt.py

```diff
@@ -0,0 +1,63 @@
+import time
```

Comment:
Please add a docstring at the top of this file briefly explaining what this code/module does

Reply:
Done :)